### ISSUE TITLE
import requests!!!

### ISSUE THREAD

**Issue by AaronZheng26** [GRRedWings/python3-android#32]
![image](https://user-images.githubusercontent.com/49259277/228285035-a0e0605e-b27b-4c1f-906d-596f4d2463d4.png)

**step 1**: Py_SetPath is deprecated, please add sys.path
`   sys.path.append("/data/user/0/com.example.pythontest/files/Python64/")
    sys.path.append("/data/user/0/com.example.pythontest/files/Python64/lib-dynload")`

**step 2**: fix collections.MutableMapping -> collections.abc.MutableMapping
maybe need cross compile again.

**Comment by nverbeek** [comment]
You've mentioned two issues here, so I'll answer them separately:

1. You are correct, `Py_SetPath` is deprecated. We have not gotten around yet to updating our sample code to use the new functions. Being that this is just a sample, you can certainly make this change in your own project. We will make a note to come back here when we are able to update the sample app.
2. The `requests` library is included in the package as a convenience. This is a 3rd party package that we do not manage. The issue here is that older requests versions reference things that have moved in later versions of Python. If you want, in your project, you can update the PythonXX.zip files to include a different version of requests that does not suffer from this issue. Alternatively, you can easily get over this in your own Python code. Prior to importing requests, you can simply put code like this:

```
if sys.version_info.major == 3 and sys.version_info.minor >= 10:
    import collections
    from collections import abc
    collections.MutableMapping = abc.MutableMapping
    collections.Mapping = abc.Mapping
    collections.Callable = abc.Callable

import requests
# rest of your code
```
The above code effectively replaces the `collections` portions at runtime to function on later versions of Python.

**Comment by GRRedWings** [comment]
Nate's thanks for the quick response on this.  I have updated the sample app to clean up the deprecated warnings.  I have also removed requests from the zip files.  While requests is useful, it really should not be included in the sample application as it is not used. As shown here, it can be confusing to think that it might be part of the python build, which it is now.

Also removing requests, allows any developer using this project to pick their own favorite http library without the potential conflict of this one.
